### PR TITLE
Prefer https secure scheme.

### PR DESCRIPTION
**Pull request by lucasmichot**
Prefer `https` secure scheme over `http` scheme.

### CODE REVIEW

File: src/Carbon/CarbonInterface.php [hunk 1]
```diff
@@ -4818,7 +4818,7 @@ public function translateTimeStringTo($timeString, $to = null);
     public static function translateWith(\Symfony\Component\Translation\TranslatorInterface $translator, string $key, array $parameters = [], $number = null): string;
 
     /**
-     * Format as ->format() do (using date replacements patterns from http://php.net/manual/fr/function.date.php)
+     * Format as ->format() do (using date replacements patterns from https://php.net/manual/en/function.date.php)
```
Comment (lucasmichot) [inline src/Carbon/CarbonInterface.php:4821]: An opportunity to change the default documentation locale too.

File: src/Carbon/Lang/bn_BD.php [hunk 2]
```diff
@@ -9,10 +9,6 @@
  * file that was distributed with this source code.
  */
 
-/*
- * Authors:
- * - Ankur Group, http://www.ankurbangla.org, http://www.bengalinux.org Taneem Ahmed, Jamil Ahmed taneem@bengalinux.org, jamil@bengalinux.org
```
Comment (lucasmichot) [inline src/Carbon/Lang/bn_BD.php:14]: Both those URLs return 404s.

File: src/Carbon/Lang/shn_MM.php [hunk 3]
```diff
@@ -11,7 +11,7 @@
 
 /*
  * Authors:
- * - ubuntu Myanmar LoCo Team http://www.ubuntu-mm.net Bone Pyae Sone bone.burma@mail.com
+ * - ubuntu Myanmar LoCo Team https://ubuntu-mm.net Bone Pyae Sone bone.burma@mail.com
```
Comment (lucasmichot) [inline src/Carbon/Lang/shn_MM.php:14]: `www.` redirects to non-www

File: src/Carbon/Lang/bn_BD.php (hunk 2 above)
Comment (mfn) [reply src/Carbon/Lang/bn_BD.php:14]: But just because the URLs don't work, there's no reason to remove the groups/names?

Comment (lucasmichot) [reply src/Carbon/Lang/bn_BD.php:14]: Re-added

File: composer.json [hunk 4]
```diff
@@ -17,11 +17,11 @@
         {
             "name": "Brian Nesbitt",
             "email": "brian@nesbot.com",
-            "homepage": "http://nesbot.com"
+            "homepage": "https://nesbot.com"
```
Comment (kylekatarnls) [inline composer.json:20]: This website does not have a valid HTTPS certificate (it maybe never did).

Hello @briannesbitt, please check if you want to update your website link and give HTTPS link if you have. Thank you.

Comment (lucasmichot) [reply composer.json:20]: Makes sense @kylekatarnls

**Review by kylekatarnls — APPROVED** [review]

**Pull request merged**